### ISSUE TITLE
[incorrect warning] Existing `reader_endpoint_address`of aws_elasticache_replication_group but Unresolved reference 

### ISSUE THREAD

**Issue by sogaoh** [VladRassokhin/intellij-hcl#327]
### Prerequisites
 - [x] Ensure you have latest version of plugin installed
 - [x] Search for possible issue duplicates

### Installation details
 - [x] IDE version : IntelliJ IDEA 2020.3.2 (Ultimate Edition) Build #IU-203.7148.57, built on January 26, 2021
 - [x] intellij-hcl plugin version: 0.7.10
 - [x] Terraform version: v0.13.6

### Terraform Configuration Files

```outputs.tf

output "out_redis_replication_group_endpoints" {
  value = [
    aws_elasticache_replication_group.redis_replication_group_module.primary_endpoint_address,
    aws_elasticache_replication_group.redis_replication_group_module.reader_endpoint_address,
  ]
}
```

### Exception
(none: Only WARNING : `Unresolved reference` )

### Expected Behavior
No WARNING `Unresolved reference` showed.

### Actual Behavior
- `Unresolved reference` showed.

<img width="823" alt="reader_endpoint_address_Unresolved_reference_20210221" src="https://user-images.githubusercontent.com/2090887/108628265-474f5100-749d-11eb-9ae3-98d6a979eba4.png">

### Steps to Reproduce
1. Generate a new `.tf` file. (aws_elasticache_replication_group) 
2. Generate a new `outputs.tf` and Copy & paste `outputs.tf` content. (See  above section `Terraform Configuration Files`)

**Comment by VladRassokhin** [comment]
Fixed in 0.7.11

**Comment by sogaoh** [comment]
Thanks for report. I've confirmed.

<img width="680" alt="スクリーンショット 2021-09-04 18 10 20" src="https://user-images.githubusercontent.com/2090887/132089235-4bfbe63f-150e-461c-b659-061e3c8a92bf.png">
